tokio/src/fs/file.rs

```diff
@@ -27,9 +27,11 @@ use std::task::Poll::*;
 /// it was opened with. Files also implement Seek to alter the logical cursor
 /// that the file contains internally.
 ///
-/// Files are automatically closed when they go out of scope.
+/// Files are automatically closed when they go out of scope, but you should always call
+/// [`flush`][flush] to ensure changes are written to disk.
 ///
 /// [std]: std::fs::File
+/// [flush]: ../io/trait.AsyncWriteExt.html#method.flush
```

Comment:
Can we use `crate::io::AsyncWriteExt::flush()` here?

Reply:
TLDR works - thanks!

I also prefer it this way, but when i was doing
`tokio::io::AsyncWriteExt::flush` a few days ago it gave me an error..
I must have done some kind of typo or something because now it works (`crate::` & `tokio::`)